src/inspector_agent.cc

```diff
@@ -401,14 +401,13 @@ void AgentImpl::ThreadCbIO(void* agent) {
 // static
 void AgentImpl::OnSocketConnectionIO(uv_stream_t* server, int status) {
   if (status == 0) {
-    inspector_socket_t* socket =
-        static_cast<inspector_socket_t*>(malloc(sizeof(*socket)));
+    inspector_socket_t* socket = new inspector_socket_t();
     ASSERT_NE(nullptr, socket);
     memset(socket, 0, sizeof(*socket));
```

Comment:
We have plans for more comprehensive refactoring, so I would prefer not to do this change for now.